target/linux/bcm5862x/base-files/etc/board.d/02_network

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+#
+# Copyright (C) 2021 OpenWrt.org
+#
+
+. /lib/functions/system.sh
+. /lib/functions/uci-defaults.sh
+
+bcm5862x_setup_interfaces()
+{
+	local board="$1"
+
+	case "$board" in
+	meraki,mx64)
+		ucidef_set_interface_lan "lan1 lan2 lan3 lan4"
+		ucidef_set_interface_wan "wan0"
+		;;
+	meraki,mx65)
+		ucidef_set_interface_lan "lan3 lan4 lan5 lan6 lan7 lan8 lan9 lan10 lan11 lan12"
+		ucidef_set_interface_wan "wan1 wan2"
+		;;
+	esac
+}
+
+bcm5862x_setup_macs()
+{
+	local board="$1"
+
+	case "$board" in
+	meraki,mx64 |\
+	meraki,mx65)
+		et0macaddr="$(get_mac_binary "/sys/bus/i2c/devices/0-0050/eeprom" 102)"
+                [ -n "$et0macaddr" ] && ucidef_set_interface_macaddr "lan" "$et0macaddr"
+		offset=1
+		;;
+	esac
+
+	# If WAN MAC isn't explicitly set, calculate it using base MAC as reference.
+	[ -z "$wan_macaddr" -a -n "$et0macaddr" ] && wan_macaddr=$(macaddr_add "$et0macaddr" $offset)
```

Comment:
No, please set this on a per-device basis. No defaults here, as vendors don't behave like that.